Cargo.toml

```diff
@@ -30,6 +30,7 @@ cargo-test-support = { path = "crates/cargo-test-support" }
 cargo-util = { version = "0.2.6", path = "crates/cargo-util" }
 cargo_metadata = "0.17.0"
 clap = "4.4.2"
+color-print = "0.3.4"
```

Comment:
> (not a blocker)
>
> The other issue is that this patch introduces one more dependency (and transitive proc-macro). This doesn't really affect Cargo development too much. However still some cargo plugins depend on cargo directly, so we still need to keep an eye on that.

syn, quote, and proc-macro2 will already be pulled in.  I can see if I can work with the maintainer to switch to syn 2 (a gix dep also pulls in syn v1) and winnow (gix pulls in nom for now).  That leaves just the macro itself which shouldn't be all that much.  While is only used in the bin for now, I'd like to work towards styled text in more places once we are using anstream.  The main issue with color-print itself is theming.  This is also balanced out with not needing to pull in the `String` feature into clap which affects binary size, compile times, and runtime performance.

So for me, I'm not too worried about it and I can easy see the situation evolving.  To what?  I don't yet know.